tcpdump.c

```diff
@@ -1305,6 +1305,10 @@ open_interface(const char *device, netdissect_options *ndo, char *ebuf)
 		if (status < 0)
 			error("%s: Can't set time stamp type: %s",
 		              device, pcap_statustostr(status));
+    else if (status == PCAP_WARNING_TSTAMP_TYPE_NOTSUP)
```

Comment:
1) The indentation appears to be different from the check for an error.

2) You might as well check for *any* status > 0, as that means "warning"; that makes it robust against future warnings being added.